raf/deno.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@effectionx/raf",
+  "version": "1.0.0-alpha.0",
+  "exports": "./mod.ts",
+  "license": "MIT",
+  "imports": {
+    "effection": "npm:effection@3.4.0"
```

Comment:
We should make this a caret dep so any 3.x release will work.

Reply:
@cowboyd unfortunately, ^3 keeps picking 3.0.3 for some reason. I don't know why.